include/dlaf/matrix/distribution.h

```diff
@@ -283,7 +287,8 @@ class Distribution {
   /// @pre 0 <= global_element < size().get<rc>().
   template <Coord rc>
   SizeType tileElementFromGlobalElement(SizeType global_element) const noexcept {
-    DLAF_ASSERT_HEAVY(0 <= global_element && global_element < size_.get<rc>(), "");
+    DLAF_ASSERT_HEAVY(0 <= global_element && global_element < size_.get<rc>(), global_element,
+                      size_.get<rc>(), );
```

Comment:
Please fix this and run bors again.

```suggestion
                      size_.get<rc>());
```